test/functional/interface_bitcoin_cli.py

```diff
@@ -25,6 +26,41 @@
 WALLET_NOT_LOADED = 'Requested wallet does not exist or is not loaded'
 WALLET_NOT_SPECIFIED = 'Wallet file not specified'
 
+
+def cli_get_info_string_to_dict(cli_get_info_string):
+    # help method to convert human readable -getinfo into a dictionary
+    cli_get_info = {}
+    lines = cli_get_info_string.splitlines()
+    line_idx = 0
+    ansi_escape = re.compile(r'(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]')
+    while line_idx < len(lines):
+        # Remove ansi colour code
+        line = ansi_escape.sub('', lines[line_idx])
+        if "Balances" in line:
+            # When Balances appears in a line, all of the following line contains "balance: wallet" until an empty line
+            cli_get_info["Balances"] = {}
+            while line_idx < len(lines) and not (lines[line_idx + 1] == ''):
+                line_idx += 1
+                balance, wallet = lines[line_idx].strip().split(" ")
+                # Remove right justification padding
+                wallet = wallet.strip()
+                if wallet == '""':
+                    # Set default wallet("") to empty string
+                    wallet = ''
+                cli_get_info["Balances"][wallet] = balance.strip()
+        elif ": " in line:
+            key, value = line.split(": ")
+            if "Balance" in key:
+                # remove Bitcoin emoji from Balance
+                key = "Balance"
+            if key == 'Wallet' and value == '""':
+                # Set default wallet("") to empty string
+                value = ''
+            cli_get_info[key.strip()] = value.strip()
+        line_idx += 1
+    return cli_get_info
```

Comment:
This seems like a lot of workaround, would it be easier to drop this and test the stdout instead? I think we do that in other tests (feature_help.py, tool_wallet.py, etc). I guess it could be done later.